### ISSUE TITLE
yolov5s dataset, validation set, test set problem

### ISSUE THREAD

**Issue by Xuan-zi-code** [ultralytics/yolov5#8294]
I would like to ask, is yolov5 trained while verifying, and if so, is the training result the result of the validation set, then if I only divide the two datasets (training set and validation set), I can put the training set Is the finished data result used as a test set, because I think some can only divide two data sets.

**Comment by glenn-jocher** [comment]
@Xuan-zi-code yes training is done on train set and validation on val set.
